src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -240,285 +246,316 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
-                  room.template as VenueTemplate
-                ) && (
-                  <>
-                    <Form.Label>{ROOM_TAXON.capital} background</Form.Label>
-                    <ImageInput
-                      onChange={changeBackgroundImageUrl}
-                      name="venue.mapBackgroundImage"
-                      setValue={setValue}
-                      register={register}
-                      small
-                      nameWithUnderscore
-                      imgUrl={
-                        roomVenue?.mapBackgroundImageUrl ??
-                        venueValues.mapBackgroundImage
-                      }
-                    />
-                    {errors?.venue?.mapBackgroundImage && (
-                      <span className="input-error">
-                        {errors?.venue?.mapBackgroundImage.message}
-                      </span>
-                    )}
-                  </>
-                )}
+            {isExternalPortal(room) ? (
+              <AdminInput
+                name="room.url"
+                autoComplete="off"
+                label={`${ROOM_TAXON.capital} url`}
+                placeholder={`${ROOM_TAXON.capital} url`}
+                register={register}
+                errors={errors}
+              />
+            ) : (
+              // NOTE: Save button doesn't work if the value is missing
+              <input name="room.url" type="hidden" ref={register} />
+            )}
 
-              {room.template &&
-                IFRAME_TEMPLATES.includes(room.template as VenueTemplate) && (
-                  <>
-                    <Form.Label>Livestream URL</Form.Label>
-                    <InputField
-                      name="venue.iframeUrl"
-                      type="text"
-                      autoComplete="off"
-                      placeholder="Livestream URL"
-                      error={errors?.venue?.iframeUrl}
-                      ref={register()}
-                    />
-                    {errors?.venue?.iframeUrl && (
-                      <span className="input-error">
-                        {errors?.venue?.iframeUrl}
-                      </span>
-                    )}
+            <AdminSection
+              title="Select the parent space for the “back” button"
+              withLabel
+            >
+              <SpacesDropdown
+                venueSpaces={venueSpaces ?? ALWAYS_EMPTY_ARRAY}
+                venueId={venueId}
+                setValue={setValue}
+                register={register}
+                fieldName="venue.parentId"
+                defaultSpace={venueValues.parentId}
+              />
+            </AdminSection>
+          </AdminSpacesListItem>
+          <AdminSpacesListItem title="Appearance" isOpened>
+            <AdminSection
+              withLabel
+              title={`${ROOM_TAXON.capital} image`}
+              subtitle="(overrides global settings)"
+            >
+              <ImageInput
```

Comment:
[NOTE]
There could be an `AdminImageInput` that wraps this `ImageInput` and the display of label and error such as it has similar DX as `AdminInput` or `PortalVisibility`.

Maybe a `@debt` comment, but kind of an awkward place here... 🤔

Reply:
added a debt comment bf032ac